squirrel/serialization/msgpack.py

```diff
@@ -65,6 +69,8 @@ def deserialize_shard_from_file(
             fs (AbstractFileSystem, optional): Filesystem to use for opening the file. If not provided, `fsspec` will
                 pick a filesystem suitable for `fp`. Defaults to None.
             mode (str): IO mode to use. Passed to :py:meth:`fs.open`. Defaults to "rb".
+            unpacker_kwarg (Dict, optional): kwargs to be passed to `msgpack.Unpacker()`.
```

Comment:
To match the case of other args.
```suggestion
            unpacker_kwarg (Dict, optional): Kwargs to be passed to `msgpack.Unpacker()`.
```